exercises/clock/canonical-data.json

```diff
@@ -1,6 +1,6 @@
 {
   "exercise": "clock",
-  "version": "1.0.1",
+  "version": "2.0.0",
   "comments": [
     "Most languages require constructing a clock with initial values,",
     "adding a positive or negative number of minutes, and testing equality",
```

Comment:
Should the comment be reworded since the proposed changes preclude the possibility of ever adding (or subtracting) negative minutes?

Reply:
Sorry, my comment appears to be redundant...